native/src/test/scala/sqs4s/internal/aws4/aws4suite/StringToSignSpec.scala

```diff
@@ -0,0 +1,55 @@
+package sqs4s.internal.aws4.aws4suite
+
+import java.io.File
+
+import cats.effect.IO
+import cats.implicits._
+import org.scalatest.Inspectors
+import sqs4s.internal.aws4._
+
+import scala.io.Source
+
+class StringToSignSpec extends IOSpec with Inspectors {
+  import common._
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/manual/d2a4u/sqs4s/pullRequest?prid=4059007)